com.unity.render-pipelines.high-definition/Documentation~/Local-Volumetric-Fog.md

```diff
@@ -41,6 +41,15 @@ The **Max Local Volumetric Fog Size** controls the maximum resolution you can us
 
 The **Max Local Volumetric Fog On Screen** controls how many Local Volumetric Fog components can appear on-screen at once. This setting also has an impact on memory.
 
+## Builtin Density Mask Texture
+
+HDRP includes 3D Density Mask Textures with different noise values and shapes that you can use in your scene. To use these Textures, import them from the HDRP samples package:
+
+1. Open the Package Manager window (menu: Window > Package Manager)
```

Comment:
Oops, "Window > Package manager" should be in bold.

1. Open the Package Manager window (menu: **Window** > **Package Manager**).